src/chrome/content/rules/ExtremeTech.com.xml

```diff
@@ -0,0 +1,32 @@
+<!--
+	Invalid certificate:
+		- discuss.extremetech.com
+		- metrics.extremetech.com, equivalent to extremetech-com.122.2o7.net
```

Comment:
Is this comment still true?

Reply:
...not sure how I missed that. Removing now.